docs/DELTA.md

```diff
@@ -20,12 +20,12 @@ const paraSwap = constructSimpleSDK(
 
 ```ts
 const DAI_TOKEN = '0x6b175474e89094c44da98b954eedeac495271d0f';
-const PSP_TOKEN = '0xcafe001067cdef266afb7eb5a286dcfd277f3de5';
+const USDC_TOKEN = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48';
```

Comment:
btw, now that we change it to USDC, it's not 18 decimals any more in the snippets down below